examples/connectivity/plot_power_connectome.py

```diff
@@ -0,0 +1,88 @@
+"""
+Extracting signals and plotting a connectome for the Power-264 seed-region atlas
+=============================================================
+
+This example shows how to extract signals from spherical seed-regions based
+on the Power-264 atlas (Power, 2011) and estimating a connectome using sparse
+inverse covariance.
+
+Power, Jonathan D., et al. "Functional network organization of the
+human brain." Neuron 72.4 (2011): 665-678.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from nilearn import datasets
+
+
+# fetch power atlas coords
+power = datasets.fetch_atlas_power_2011()
+power_coords = power.rois[['x', 'y', 'z']]
+
+#fetch dataset
+adhd = datasets.fetch_adhd(n_subjects=1)
+
+# Preprocess image and extract signals from Power-264
+from nilearn import input_data
+
+masker = input_data.NiftiSpheresMasker(seeds=power_coords,
+                                       radius=5.,
+                                       standardize=True,
+                                       smoothing_fwhm=4,
+                                       detrend=True,
+                                       low_pass=0.1,
+                                       high_pass=0.01,
+                                       t_r=2.5,
+                                       verbose=5)
+
+timeseries = masker.fit_transform(adhd.func[0], confounds=adhd.confounds[0])
+
+# calculate connectivity and plot Power-264 correlation matrix
+connectivity = np.arctanh(np.corrcoef(timeseries.T))
```

Comment:
That is a lot of steps in one line for a tutorial example. In particular, perhaps do the ``arctanh`` in a separate line with a comment why you do this scaling.